How We Work updates (#76)
OHS-Hosting-Infrastructure/infrastructure: docs/how-we-work/README.md

* **Scheduled maitenence windows** outside of core working hours occur once a month, every **3rd Thursday of the month at 8pm EST**.
* Critical requests, such as infrastructure outages, should be addressed as soon as possible even if it is outside of core business hours.

## Conventions

* Use labels to identify type of ticket
  * Bug: defect that needs to be fixed
  * Task: change that delivers value to client (i.e. application team or end user)
  * Chore: change that delivers value to team, but not a direct value to client (i.e. lets team work better or easier)
 * Use additional labels for categorization
   * Documentation: when a ticket is only for documentation
   * Maintenance: when a ticket is to manage a maintenance window
   * etc.
 * Ticket estimation:
   * Items are estimated using Fibonnaci points of 1, 2, 3, and 5
     * If any item is estimated as 5, needs to be broken down

>      * 1: Well-understood and can be completed without requiring interaction between teams in less 1-2 work hours

Reviewer: ```suggestion
     * 1: Well-understood and can be completed without requiring interaction between teams in less than 1-2 work hours
```
Sorry that's my b

     * 3: Some ambiguity or well understood but requires interaction with other teams that may delay completion, likely would take 1 full work day
   * Point estimates are put in parenthesis at end of ticket title
   * All team members vote on points, whether or not they're in a technical role 
 * Epics or high level initiative name is put in square brackets at beginning of ticket title
 * Tickets that are blocked by other tickets are identified in description with link to blocking ticket
 * Bugs are not estimated nor are maintenance window items

## Sprint structure

The Headstart Hosting team works on a one week sprint cadence with the following structure:

* Sprint wrap/planning
* Sprint grooming
* Daily Standups
* Demo
* Retrospective

Our sprints start every Tuesday after Sprint Planning.

TODO: Note the next few sprint starts.

## Meetings

### __Sprint Grooming__

This meeting will occur once a sprint.

We will groom tickets in the backlog to make sure they are ready to review for sprint planning.

Grooming activities include:
* Adjusting placement of tickets in backlog based on priority
* Ticket clarification ensuring that Acceptance Criteria is clear

> * Estimation of tickets and splitting of tickets if needed

Reviewer: What's estimation of tickets? That's not pointing, is it?

Author: Yes, pointing.

Reviewer: We don't do pointing in grooming.

Reviewer: We COULD split them, though.

Author: hmm...so we only point during sprint planning? it helps provide visibility for the work in backlog if more tickets than just what's for next sprint are estimated.

Reviewer: I've always done that way. Happy to try out this suggestion, but then we have the issue of everyone having to be at both grooming and sprint planning. Generally I like to keep grooming smaller. What do you think?

Author: As discussed after standup, it's helpful for full team to be present at both meetings.

* **Scheduled maitenence windows** outside of core working hours occur once a month, every **3rd Thursday of the month at 8pm EST**.
* Critical requests, such as infrastructure outages, should be addressed as soon as possible even if it is outside of core business hours.

## Conventions

* Use labels to identify type of ticket
  * Bug: defect that needs to be fixed
  * Task: change that delivers value to client (i.e. application team or end user)
  * Chore: change that delivers value to team, but not a direct value to client (i.e. lets team work better or easier)
 * Use additional labels for categorization
   * Documentation: when a ticket is only for documentation
   * Maintenance: when a ticket is to manage a maintenance window
   * etc.
 * Items are estimated using Fibonnaci points of 1, 2, 3, and 5
   * If any item is estimated as 5, needs to be broken down

>    * 1: Well-understood and can be completed without requiring interaction between teams within 1 day

Reviewer: ```suggestion
   * 1: Well-understood and can be completed without requiring interaction between teams in less 1-2 work hours
```

>    * 2: Well-understood and can be completed with little to no interaction between teams in a few days

Reviewer: ```suggestion
   * 2: Well-understood and can be completed with little to no interaction between teams within half a day of concentrated work
```

>    * 3: Some ambiguity or well understood but requires interaction with other teams that may delay completion

Reviewer: ```suggestion
   * 3: Some ambiguity or well understood but requires interaction with other teams that may delay completion, likely would take 1 full work day
```

 * Point estimates are put in parenthesis at end of ticket title

>  * Epics or high level initiative name is put in square brackets at beginning of ticket title

Reviewer: ```suggestion
  * All team members vote on points, whether or not they're in a technical role. 
  * Epics or high level initiative name is put in square brackets at beginning of ticket title
```
